docs/quickstart.rst

```diff
@@ -203,23 +203,17 @@ Getting Started with Docker Client
 
     The following steps show each Docker container being launched in detached mode and how to access the logs for those detached containers.  If you prefer to run the containers in the foreground, you can do so by replacing the ``-d`` ("detached") flag with ``--it`` ("interactive"). Containers launched in interactive mode will stream the log messages for the Confluent service directly to the terminal window.  For that reason, you'll need a separate terminal for each Docker image launched in interactive mode.
 
-Create and configure the Docker Machine. This example creates a VirtualBox VM with ~6 GB of memory to serve as your Docker host.
+Docker Network
+++++++++++++++
 
-  .. sourcecode:: console
-
-    $ docker-machine create --driver virtualbox --virtualbox-memory 6000 confluent
-
-  .. note::
-
-    If you want to reuse a Docker Machine you've already created under the name "confluent" run: ``docker-machine start confluent`` instead.
+First, create the network to run the Confluent containers in.
```

Comment:
suggest: `Create the Docker network that is used to run the Confluent containers in.`